pages/documentation/developer/tutorials/diagram/svg-writing.md

```diff
@@ -177,10 +177,10 @@ We customize a bit further the parameters: the feeder names in this substation a
 
 ```java
 // Customize further the parameters to have the feeders label rotated, in order to avoid overlapping
-layoutParameters.setLabelDiagonal(true);
+svgParameters.setLabelDiagonal(true);
 
 // Draw the diagram of substation PP_Amsterdam (id _c49942d6-8b01-4b01-b5e8-f1180f84906c)
-SingleLineDiagram.draw(network, "c49942d6-8b01-4b01-b5e8-f1180f84906c", Paths.get("/tmp/AmsterdamSubstation.svg"), layoutParameters);
+SingleLineDiagram.draw(network, "c49942d6-8b01-4b01-b5e8-f1180f84906c", Paths.get("/tmp/AmsterdamSubstation.svg"), sldParameters);
 ```
 
 We then obtain the following SVG file representing the whole PP_Amsterdam substation with its three voltage levels:
```

Comment:
Did you update the SVG files corresponding to this page? I think there are some changes at least in the CSS.